### ISSUE TITLE
DEPR/TEST: inspect deprecation warning

### ISSUE THREAD

**Issue by jreback** [pydata/pandas#12101]
hmm, this looks pretty new. should fix this code.

```
...../home/travis/build/pydata/pandas/pandas/tests/test_panel.py:201: DeprecationWarning: inspect.getargspec() is deprecated, use inspect.signature() instead
```

build is [here](https://travis-ci.org/pydata/pandas/jobs/103526993). This is a numpy-dev build, need to check out the other builds.

**Comment by ranarag** [comment]
please replace with attached file and check whether the problem is fixed
(P.S. before replacing please 
[test_panel.txt](https://github.com/pydata/pandas/files/98005/test_panel.txt)
change the .txt extension to .py)

**Comment by jreback** [comment]
closed by #12102
